src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -67,7 +67,7 @@ export class GithubAppsLogsComponent implements OnInit {
     this.columnsToDisplay = this.matDialogData.userId
       ? ['organization', 'repository', 'reference', 'success', 'type']
       : ['repository', 'reference', 'success', 'type'];
```

Comment:
Not sure if others feel the same, but I prefer if the entry name is after the repository column to stay consistent with our entry path format of organization/repository/entry-name